core/ir/aarch64/codec.c

```diff
@@ -2264,6 +2266,78 @@ encode_opnd_x0p1(uint enc, int opcode, byte *pc, opnd_t opnd, OUT uint *enc_out)
     return encode_opnd_wxnp(true, 1, 0, opnd, enc_out);
 }
 
+static inline bool
+decode_opnd_x0p2(uint enc, int opcode, byte *pc, OUT opnd_t *opnd)
+{
+    return decode_opnd_wxnp(true, 2, 0, enc, opnd);
```

Comment:
suggestion: Add param names: so `/*is_x=*/true, /*plus=*/2, /*pos=*/0, enc, opnd` here and below to make it easier for the reader.